src/main/groovy/com/smartbear/swagger/Swagger2Importer.groovy

```diff
@@ -138,7 +138,7 @@ class Swagger2Importer implements SwaggerImporter {
 
         RestMethod method = resource.addNewMethod(opName)
         method.method = httpMethod
-        method.description = operation.summary
+        method.description = (operation.description ?: "").concat(System.getProperty("line.separator")).concat(operation.summary ?: "")
```

Comment:
Definitely no in this case. Even more, we can't be sure that our separator (current or new one) will not be presented in the description or/and summary.
I don't know how frequently this case will happen. May be we can ignore it if it will be too rare?